docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Do not use this document to release changes for previous major versions. We have not tested the document in this scenario, and extra work would be needed to tell NPM not to mark the release as the 'latest'
+
+Follow this document to publish a patch release of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix:
+
+- as part of a patch release, after the team has started to merge changes for a new feature release into the `main` branch - for example, a 3.14.x release once we've started merging changes for 3.15.0
+- as part of a release, after the team has started to merge changes for a new breaking release into the `main` branch - for example, a 3x.x release once we've started merging changes for 4.0.0
+
+If the `main` branch contains unreleased changes that are releasable immediately, [follow the steps in Publishing GOV.UK Frontend](/docs/releasing/publishing.md) instead.
+
+If the `main` branch only has a few unreleasable changes and these are easy to revert, you can do the following:
+
+1. Revert the unreleasable changes on the `main` branch
+2. Follow the steps to publish GOV.UK Frontend
+3. Add the reverted changes to the `main` branch
+
+However, this approach has risks. For example, it creates a messy commit history on the `main` branch.
+
+## Release GOV.UK Frontend from support branch
```

Comment:
Given that this section explicitly speaks to different parties, it might be worth defining who this content is for overall at the start. Who is following this content? Is it anyone in the Design System team? Or is it a specific person who then has to notify the other parties mentioned in this section what they need to do?

Reply:
Addressed in suggested wording for line 3. :)